src/qtt/gui/live_plotting.py

```diff
@@ -62,48 +68,59 @@ def __init__(self):
             print('rda_t: check whether redis is installed and the server is running')
             raise e
 
-    def get_float(self, key, default_value=None):
-        """ Get value by key and convert to a float """
+    def get_float(self, key: str, default_value: Optional[float] = None) -> Optional[float]:
+        """ Get value by key and convert to a float
+
+        Args:
+            key: Key to retrieve
+            default_value: Value to return when the key is not present
+
+        Returns:
+            Value retrieved
+        """
         v = self.get(key, default_value)
         if v is None:
             return v
         return float(v)
 
-    def get_int(self, key, default_value=None):
+    def get_int(self, key: str, default_value: Optional[int] = None) -> Optional[int]:
         """ Get value by key and convert to an int
 
+        Args:
+            key: Key to retrieve
+            default_value: Value to return when the key is not present
+
         Returns:
-            value (int)
+            Value retrieved
         """
         v = self.get(key, default_value)
         if v is None:
             return v
         return int(float(v))
 
-    def get(self, key, default_value=None):
+    def get(self, key: str, default_value: Optional[Any] = None) -> Any:
```

Comment:
The returning type should be Optional[Any]. When value is None and default_value is None (the default value of default value) it returns None.

Reply:
Is this only to make it symmetric with the other return values? Because `Any` includes a `None` right? 

I updated the annotation as suggested

Reply:
Optional[T] is just an alias for Union[T, None].
Union[Any, None] isn't the same as Any (https://github.com/python/mypy/issues/3138)